assets/scripts.js

```diff
@@ -20,4 +19,79 @@ $(function () {
       listUrls.addClass('is--visible');
     }
   });
-})
+
+  // Adding copy buttons to all pre tags parentNode and add class code-container
+  var preBlocks = document.getElementsByTagName('pre');
```

Comment:
Shouldn't this be used on the `code` tag instead of `pre` tag, what do you think?

Reply:
No because, pre and code tags are different.
We have simple code also like this `code`
Whereas the ones enclosed in pre are like this one
```python
python code.py
```
So, that's why pre is used.

Reply:
Inspect this above comment...

Reply:
Oh great, fine, keep it this way :+1: